README.md

```
@@ -68,45 +39,34 @@ or remove types:
 
 - `die: dN` — optional; sets the die to one of `d4, d6, d8, d10, d12, d20, d100`.
-  Omit it to default to **d20**. Declare it at most once. You can also roll a
-  **pool** — `die: 2d6`, `die: 3d10` — where the middle sums are more likely
-  than the extremes.
-- Every other non-blank line is one result. Prefix a line with `Nx ` (e.g.
-  `3x Goblin ambush`) to weight it.
-- A result can be a `[[wikilink]]` — e.g. to nest another table.
+  Omit it to default to d20.
+  You can also roll a pool — `die: 2d6`, `die: 3d10` — where the middle sums are more likely than the extremes.
+- Every other non-blank line is one result.
+  Prefix a line with `Nx ` (e.g. `3x Goblin ambush`) to weight it.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)